src/components/shell/shell.js

```diff
@@ -214,6 +214,9 @@ class Shell extends Component {
     const {isIPadOS, isTablet, isMobile} = player.env;
     this.props.updateIsMobile(isIPadOS || isTablet || isMobile || forceTouchUI);
     this._onWindowResize();
+    window.addEventListener('resize', () => {
```

Comment:
also see https://github.com/kaltura/playkit-js-ui/pull/367 and verify this issue with your change

Reply:
use the event manager for this as well

Reply:
done